.github/workflows/push.yml

```diff
@@ -11,6 +11,8 @@ on:
       - release/v*
     tags:
       - 'v*'
+  schedule:
```

Comment:
The `push` workflow is pretty heavy and does much more than performance test. E.g. it produces / signs docker images and binaries. Maybe we could extract a separate workflow just for performance tests?

Reply:
Ok, I can do that, but this workflow runs on every code push to `master` or `release`. Isn't it better to change its trigger also then? Maybe when a pull request on those branches is closed and merged?